examples/vulkan/Vulkan.cpp

```diff
@@ -462,7 +461,7 @@ class VulkanExample
     void setupInstance()
     {
         // Load bootstrap entry points
-        gladLoadVulkan(0, getVulkanFunction);
+        gladLoadVulkan({}, getVulkanFunction);
```

Comment:
Would `nullptr` make sense here?

Reply:
This is tricky because for whatever reason, the VS 2019 jobs fail if we try to use `nullptr` because in that configuration, these functions expect an integer instead of a pointer. I'm not sure why this is the case. `0` works since that is a valid pointer and integer value as is `{}` but using `{}` means we silence this clang-tidy check. If we go back to using zero then we just have to disable this check for this file.

Reply:
☹️